evennia/contrib/rpg/rpsystem/tests.py

```diff
@@ -96,7 +96,7 @@ def test_obfuscate_whisper(self):
 recog02 = "Mr Receiver2"
 recog10 = "Mr Sender"
 emote = 'With a flair, /me looks at /first and /colliding sdesc-guy. She says "This is a test."'
-case_emote = "/me looks at /first, then /FIRST, /First and /Colliding twice."
+case_emote = "/Me looks at /first, then /FIRST, /First and /Colliding twice."
```

Comment:
This seems like a regression? You should be able to use `/me` in every situation?

Reply:
The original code had `/me` explicitly preserving case every time, but I removed the exception.

I can guess at the logic behind preserving case, but it only holds up from the perspective of the sender - since they always see their own key. If you did e.g. the first emote test with case sensitivity but excluding the case sensitivity for self-references, the output to Receiver1 would look like this:
```
"With a flair, |bA nice sender of emotes|n looks at |mReceiver1|n and "
'|banother nice colliding sdesc-guy for tests|n. She says |w"This is a test."|n',
```

And if your sdesc was `a nice sender of emotes` and you tried to explicitly capitalize with `/Me` like you would for the other references, it wouldn't work.

Reply:
Ok, I can see that argument. I would then strongly suggest to also add unit tests for `/me` and `/Me` respectively.